doc/devel/contribute.rst

```diff
@@ -161,6 +161,29 @@ please reach out on the :ref:`contributor_incubator`
 
 .. _`open an issue`: https://github.com/matplotlib/matplotlib/issues/new?assignees=&labels=Documentation&projects=&template=documentation.yml&title=%5BDoc%5D%3A+
 
+.. _contribute_triage:
+
+Triage
+------
+The `issue tracker <https://github.com/matplotlib/matplotlib/issues>`_ is our
+centralized location for feature requests, bug reports,
+tracking major projects and discussing priorities. It takes a lot of
+time and effort to keep oragnized, so curating our issue tracker is a valuable
+contribution. Some examples are:
+
+* labeling issues and pull requests
+* verifying bug reports
+* debugging and resolving issues
+* linking to related issues, discussion, and external work
+
+Helping out with triage is discussed in detail in :ref:`bug_triaging`.
```

Comment:
As with code and documentation we should link the Contributor incubator. Code / documentation says

> If you get stuck, please reach out on the [Contributor incubator](https://matplotlib.org/devdocs/devel/contribute.html#contributor-incubator)

Likely that's not exactly the best the wording for triage. Which process do we want exactly? Is it ok that people just start to comment on issues? Or should they first contact us through the incubator.

Reply:
Maybe something like

> If you have any questions about the process, please reach out on...

If someone is stuck about how to triage a particular issue, it's maybe better they just leave it for someone else to pick up (at least, that's what I do...!)

I think it's fine if people just start commenting on issues.  We are not overrun with people wanting to do this so I think it's pretty easy to keep track of what anyone new is doing.  We do not require people to ask first when making other contributions like creating PRs.